src/ngraph/frontend/onnx_import/op/softplus.cpp

```diff
@@ -37,34 +29,11 @@ namespace ngraph
             {
                 NodeVector softplus(const Node& node)
                 {
-                    auto data = node.get_ng_inputs().at(0);
-
-                    std::shared_ptr<ngraph::Node> zero_node =
-                        std::make_shared<ngraph::op::Constant>(
-                            data->get_element_type(), data->get_shape(), std::vector<float>{0.f});
-                    std::shared_ptr<ngraph::Node> one_node = std::make_shared<ngraph::op::Constant>(
-                        data->get_element_type(), data->get_shape(), std::vector<float>{1.f});
-
-                    std::shared_ptr<ngraph::Node> positive_val_node =
-                        data + std::make_shared<ngraph::op::Log>(
-                                   std::make_shared<ngraph::op::Exp>(
-                                       std::make_shared<ngraph::op::Negative>(data)) +
-                                   one_node);
-
-                    std::shared_ptr<ngraph::Node> negative_val_node =
-                        std::make_shared<ngraph::op::Log>(std::make_shared<ngraph::op::Exp>(data) +
-                                                          one_node);
+                    const auto data = node.get_ng_inputs().at(0);
 
-                    std::shared_ptr<ngraph::Node> condition_node =
-                        std::make_shared<ngraph::op::Greater>(data, zero_node);
+                    const auto fused_softplus = std::make_shared<ngraph::op::Softplus>(data);
 
-                    //
-                    // This equation represents:
-                    //     x + log(exp(-x) + 1) - for x > 0; to manage exponent overflow,
-                    //     log(exp(x) + 1)      - elsewhere.
-                    //
-                    return {std::make_shared<ngraph::op::Select>(
-                        condition_node, positive_val_node, negative_val_node)};
+                    return fused_softplus->decompose_op();
```

Comment:
Decomposing a fused op in the ONNX importer would defeat the purpose of creating the fused op.
ONNX importer must output fused op without decomposing them.